stress/v2/statement/go.go

```diff
@@ -0,0 +1,35 @@
+package statement
+
+import (
+	"fmt"
+
+	"github.com/influxdata/influxdb/stress/v2/ponyExpress"
+)
+
+// GoStatement is a Statement Implementation to allow other statements to be run concurrently
+type GoStatement struct {
+	Statement
+
+	StatementID string
+}
+
+// SetID statisfies the Statement Interface
+func (i *GoStatement) SetID(s string) {
+	i.StatementID = s
+}
+
+// Run statisfies the Statement Interface
+func (i *GoStatement) Run(s *ponyExpress.StoreFront) {
```

Comment:
The `GO` statement allows you to run multiple concurrent statements.  In the example below

```
INSERT foo
measurement,
tag=[val1|val2]
field=[int inc(0) 1000]
100000 10s

INSERT bar
measurement1,
tag=[val1|val2]
field=[int inc(0) 1000]
100000 10s
```

The `INSERT` statements will run synchronously. In the below example they will run concurrently:

```
GO INSERT foo
measurement,
tag=[val1|val2]
field=[int inc(0) 1000]
100000 10s

GO INSERT bar
measurement1,
tag=[val1|val2]
field=[int inc(0) 1000]
100000 10s

WAIT
```